modules/react/table/index.ts

```diff
@@ -1 +1,3 @@
 export * from './lib/Table';
+export * from './lib/BaseTable';
+export * from './lib/parts';
```

Comment:
i think you also want to add `TableRow` since that folder doesn't have an `index.tsx`

Reply:
Yup added an index file so I can export from the folder.